releasenotes/notes/features-88f5ec5ec6b5c380.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    Enabling the option to strip the arguments of the commands collected by the process Agent on Windows.
```

[minor] case / also is this a feature or a bug fix?
```suggestion
    Enabling the option to strip the arguments of the commands collected by the process agent on Windows.
```